css/main.css

```diff
@@ -0,0 +1,1014 @@
+  margin-top: -10px;
+  color: rgba(39, 42, 49, 0.7803921569);
+  font-size: 15px;
+  padding: 0 40px;
+}
+
+.last-Event-container {
+  display: grid;
+  gap: 50px;
+  padding: 30px;
+}
+
+.last-Event-container .last-Event-item:nth-child(1),
+.last-Event-container .last-Event-item:nth-child(2) {
+  height: 300px;
+  width: 100%;
+}
+
+.last-Event-container .last-Event-item:nth-child(1) {
+  height: 300px;
+  width: 100%;
+}
+
+.last-Event-container .last-Event-item:nth-child(1) .Event-year-location {
+  background-color: #ec5242;
+  height: 300px;
+  color: #fff;
+  padding: 0;
+  margin-bottom: -300px;
+  mix-blend-mode: multiply;
+}
+
+.last-Event-container .last-Event-item:nth-child(1) .Event-resources-1 {
+  width: inherit;
+  height: inherit;
+  display: flex;
+}
+
+.last-Event-container .last-Event-item:nth-child(2) {
+  height: 300px;
+  width: 100%;
+}
+
+.last-Event-container .last-Event-item:nth-child(2) .Event-year-location-2 {
+  background-color: #ec5242;
+  height: 300px;
+  margin-bottom: -15px;
+  padding: 0;
+  mix-blend-mode: multiply;
+}
+
+.last-Event-container .last-Event-item:nth-child(2) .Event-resources-2 {
+  width: inherit;
+  height: inherit;
+  margin-top: -300px;
+}
+
+.contact-year-text {
+  margin-top: -190px;
+  width: 100%;
+}
+
+.contact-year-text-head {
+  color: #fff;
+  font-size: 1.5rem;
+  position: relative;
+  text-align: center;
+  padding: 0 40px;
+  font-weight: 400;
+}
+
+.contact-year-text-info {
+  position: relative;
+  color: #fff;
+  font-size: 15px;
+  text-align: center;
+  padding: 0 50px;
+  font-weight: 300;
+}
+
+.partners {
+  background-image: linear-gradient(250deg, #272a31, rgba(39, 42, 49, 0.7803921569));
+}
+
+.partners .partner-head {
+  text-align: center;
+  color: #fff;
+  padding-top: 40px;
+  font-weight: 500;
+  font-size: 25px;
+  line-height: 20px;
+}
+
+.partners .partner-head-red-line {
+  color: #ec5242;
+  text-align: center;
+  margin-top: -25px;
+  font-weight: 300;
+}
+
+.partners .partners-logo-ul {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 40px 40px 40px;
+}
+
+.partners .partners-logo-ul .partner-logo {
+  max-width: 80px;
+  height: auto;
+}
+
+.footer-end {
+  display: flex;
+  flex-direction: row;
+}
+
+.footer-end .footer-end-items {
+  width: 50%;
+  padding: 40px;
+  text-align: center;
+}
+
+.footer-end .footer-end-items .footer-end-items-logo {
+  font-weight: 700;
+  line-height: 30px;
+  color: #272a31;
+}
+
+.footer-end .footer-end-items .footer-end-items-rights {
+  font-weight: 400;
+  font-size: 15px;
+  color: rgba(39, 42, 49, 0.7803921569);
+}
+
+.hamburger-pop-up {
+  position: fixed;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  background-color: #fff;
+  mix-blend-mode: multiply;
+  display: none;
+  margin-left: -20px;
+  -webkit-backdrop-filter: blur(20px);
+  -moz-backdrop-filter: blur(20px);
+  -o-backdrop-filter: blur(20px);
+  backdrop-filter: blur(20px);
+  border-radius: 5px;
+
+  /* overflow: hidden; */
+}
+
```

Comment:
- [ ] Kindly delete any commented code as it conflicts with the best practices and having a clean code.

Reply:
Done 👍 @Ofarouq310